src/app/component/core/nav-bar/nav-bar.component.ts

```diff
@@ -83,6 +84,10 @@ export class NavBarComponent implements OnInit {
     });
   }
 
+  public openModalSignIn(){
+      this.dialog.open(WindowsigninComponent);
+    }
+
   openDialogProposeCafeComponent(): void {
```

Comment:
Please add access modifiactor (private or public)

Reply:
pay attention, this code does not apply to my forms.